backend/tests/test_model_utils.py

```diff
@@ -1,13 +1,18 @@
 """Tests for the utils module"""
 
 import unittest
-
-from models.utils.generic_utils import nhs_number_mod11_check
+from copy import deepcopy
+from models.utils.generic_utils import nhs_number_mod11_check, get_nhs_number_verification_status_code
+from .utils.generic_utils import load_json_data
 
 
 class UtilsTests(unittest.TestCase):
     """Tests for models.utils.generic_utils module"""
 
+    @classmethod
+    def setUpClass(cls):
+        cls.json_data = load_json_data("sample_covid_immunization_event.json")
+
```

Comment:
Is this code being used anywhere? The tests in this class aren't using any JSON data